### PR TITLE
Add installation docs

### PR DESCRIPTION
**Pull request by guineveresaenger**
Part of https://github.com/pulumi/home/issues/3598

Specifically built against https://github.com/pulumi/pulumi-terraform-bridge/pull/2349.

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -217,13 +218,39 @@ func Provider() tfbridge.ProviderInfo {
...
+				return b, nil
+			},
 		},
-	})
+		removeTutorialLink,
+		removeTFOrLater,
+	)
+}
+
+var tutorialLink = "Try the hands-on tutorial on the Datadog provider on the HashiCorp Learn site."
```
Comment (iwahbe) [inline provider/resources.go:236]:
```suggestion
const tutorialLink = "Try the hands-on tutorial on the Datadog provider on the HashiCorp Learn site."
```

File: provider/resources.go [hunk 2]
```diff
@@ -217,13 +218,39 @@ func Provider() tfbridge.ProviderInfo {
...
+
+// Removes a reference to Consul Remote State Backend
+var removeTutorialLink = tfbridge.DocsEdit{
+	Path: "index.md",
+	Edit: func(_ string, content []byte) ([]byte, error) {
+		before, after, _ := bytes.Cut(content, []byte(tutorialLink))
+		content = append(before, after...)
```
Comment (iwahbe) [inline provider/resources.go:243]: `bytes.Cut` & `append` will just remove the first instance of `tutorialLink`. Should we use `bytes.Split` & `bytes.Join` instead?

File: provider/resources.go [hunk 3]
```diff
@@ -217,13 +218,39 @@ func Provider() tfbridge.ProviderInfo {
...
+		return content, nil
+	},
+}
+
+// Removes a reference to TF version and compatibility
+var TFVersionOrLaterRegexp = regexp.MustCompile(`It requires [tT]erraform [0-9]+\.?[0-9]?\.?[0-9]? or later.`)
+var removeTFOrLater = tfbridge.DocsEdit{
+	Path: "index.md",
+	Edit: func(_ string, content []byte) ([]byte, error) {
+		content = TFVersionOrLaterRegexp.ReplaceAllLiteral(content, nil)
```
Comment (iwahbe) [inline provider/resources.go:253]: This variable shouldn't be imported, so let's keep it private.
```suggestion
var tfVersionOrLaterRegexp = regexp.MustCompile(`It requires [tT]erraform [0-9]+\.?[0-9]?\.?[0-9]? or later.`)
var removeTFOrLater = tfbridge.DocsEdit{
	Path: "index.md",
	Edit: func(_ string, content []byte) ([]byte, error) {
		content = tfVersionOrLaterRegexp.ReplaceAllLiteral(content, nil)
```

File: provider/resources.go [hunk 4]
```diff
@@ -217,13 +218,39 @@ func Provider() tfbridge.ProviderInfo {
...
+	Path: "index.md",
+	Edit: func(_ string, content []byte) ([]byte, error) {
+		before, after, _ := bytes.Cut(content, []byte(tutorialLink))
+		content = append(before, after...)
+		return content, nil
+	},
+}
+
+// Removes a reference to TF version and compatibility
+var TFVersionOrLaterRegexp = regexp.MustCompile(`It requires [tT]erraform [0-9]+\.?[0-9]?\.?[0-9]? or later.`)
```
Comment (iwahbe) [inline provider/resources.go:249]: This pass makes perfect sense. Should we add it to the default transformation rules?

**Review by iwahbe — APPROVED** [review]

File: provider/resources.go (hunk 4 above)
Comment (guineveresaenger) [reply provider/resources.go:249]: https://github.com/pulumi/pulumi-terraform-bridge/pull/2349

File: provider/resources.go (hunk 3 above)
Comment (guineveresaenger) [reply provider/resources.go]: Removed as this is now in the defaults in the bridge installation-docs transformations.

**Pull request merged**